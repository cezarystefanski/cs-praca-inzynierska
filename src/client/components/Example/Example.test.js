import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Example from './Example.react';

const exampleMock = renderer
  .create((
    <MemoryRouter>
      <Example
        exampleStoreNumber={500}
        onBindClick={() => {}}
        onIncreaseClick={() => {}}
        onDecreaseClick={() => {}}
      />
    </MemoryRouter>
  ))
  .toJSON();

describe('home component', () => {
  it('renders correctly', () => {
    expect(exampleMock).toMatchSnapshot();
  });
});
