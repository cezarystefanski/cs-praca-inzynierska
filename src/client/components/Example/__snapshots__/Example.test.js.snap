// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home component renders correctly 1`] = `
<div>
  <p>
    New example number is 
    <span
      className="exampleNumber"
    >
      500
    </span>
    <button
      onClick={[Function]}
    >
      Bind to store
    </button>
  </p>
  <p>
    <small>
      Store number is 
      <span
        className="exampleNumber"
      >
        500
      </span>
    </small>
    <button
      onClick={[Function]}
    >
      -
    </button>
    <button
      onClick={[Function]}
    >
      +
    </button>
  </p>
  <a
    href="/"
    onClick={[Function]}
  >
    Link to home
  </a>
</div>
`;
